common-files/Jenkinsfile

```diff
@@ -0,0 +1,11 @@
+/*
+ See the documentation for more options:
+ https://github.com/jenkins-infra/pipeline-library/
+*/
+def recentLTS = "2.222.1"
```

Comment:
This will clash with #172. Frankly I am inclined to discourage use of the `jenkins` parameter generally and just recommend maintainers keep `jenkins.version` relatively new, while moving towards wider use of PCT.

Reply:
Removed it due to clash